docs/components/modeler/desktop-modeler/connect-to-camunda-cloud.md

```diff
@@ -18,8 +18,12 @@ Desktop Modeler can directly deploy diagrams and start process instances in Camu
 
 ![deployment via Camunda Platform 8](./img/deploy-diagram-camunda-cloud-remember.png)
 
-4. Select the **Remember** checkbox if you want to locally store the connection information.
+4. Click the **Remember credentials** toggle to locally store the connection information.
```

Comment:
```suggestion
4. Toggle on **Remember credentials** to locally store the connection information.
```

Reply:
Adjusted to `Toggle **Remember credentials** to locally store the connection information.`